docs/tutorial/local_search.ipynb

```diff
@@ -0,0 +1,497 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4402d3c8-5632-40be-8e4e-523648aea7ae",
+   "metadata": {},
+   "source": [
+    "# Local Search Algorithm in Qamomile\n",
```

Comment:
Local search is intended to be used as a post-processing step, so it should be a tutorial that explains how to use it in that way.
For example, the problem in the QRAO tutorial should work with VQE up to about 24 nodes, so I think it would be good to apply local search to the state obtained by Pauli Rounding.
https://github.com/Jij-Inc/Qamomile/blob/main/docs/tutorial/qrao_tutorial.ipynb

Reply:
Maybe this should belong to a new PR?

Reply:
If so, could you remove the current version of the tutorial?

Reply:
Done